ta/arch/arm/link.mk

```diff
@@ -5,24 +5,24 @@ link-script-dep$(sm) = $(link-out-dir$(sm))/.ta.ld.d
 SIGN = $(ta-dev-kit-dir$(sm))/scripts/sign.py
 TA_SIGN_KEY ?= $(ta-dev-kit-dir$(sm))/keys/default_ta.pem
 
-all: $(link-out-dir$(sm))/$(binary).dmp \
-	$(link-out-dir$(sm))/$(binary).stripped.elf \
-	$(link-out-dir$(sm))/$(binary).ta
-cleanfiles += $(link-out-dir$(sm))/$(binary).elf
-cleanfiles += $(link-out-dir$(sm))/$(binary).dmp
-cleanfiles += $(link-out-dir$(sm))/$(binary).map
-cleanfiles += $(link-out-dir$(sm))/$(binary).stripped.elf
-cleanfiles += $(link-out-dir$(sm))/$(binary).ta
+all: $(link-out-dir$(sm))/$(user-ta-uuid).dmp \
```

Comment:
Ah, now I get it ;-).
Still, it looks like we have a backward compatibility problem with the parent commit. Just a missing `user-ta-uuid := $(binary)` I would say. Or even better, do not introduce `$(user-ta-uuid)` in the parent commit.

Reply:
True, I'll move the `user-ta-uuid` stuff into this commit.